tests/ert_tests/ert3/evaluator/test_evaluator.py

```diff
@@ -75,7 +80,7 @@ def test_evaluator_script(
         step_builder,
     )
 
-    if isinstance(stage, ert3.config.Unix):
```

Comment:
Isn't it better to keep the original check here, so we have one less parameter to the test (as this is directly dependent on the config)?

Reply:
I would argue no - because forking in tests can quickly become obsolete, as was the example in the test I deleted, where the `if` was never actually used. That could become the effect here, potentially without us noticing the behavoir changed (albeit unlikely, it could happen).